rosdep/python.yaml

```diff
@@ -701,6 +701,14 @@ python-falcon:
   fedora: [python-falcon]
   ubuntu: [python-falcon]
 python-fcn-pip:
+  debian:
+    pip:
+      depends: [libhdf5-dev, liblapack-dev]
+      packages: [fcn]
+  fedora:
+    pip:
+      depends: [hdf5-devel, lapack-devel]
```

Comment:
@k-okada they should be rosdep key names, they look a lot like the debian names since that's our default, but even for fedora I think it should be the same libhdf5-dev, liblapack-dev ala https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L1798 and https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L1905